src/components/new-members/index.js

```diff
@@ -4,8 +4,9 @@ import styles from '@components/new-members/new-members.module.scss';
 import { membersContext } from '@store/members/members-context';
 import { searchMemberContext } from '@store/search-members/searchMembers-context';
 import { searchMembers } from '@helper-functions/search-members';
-import Link from 'next/link';
 import { userContext } from '@store/user/user-context';
+import { useRouter } from 'next/router';
+import { useKeyboardContext } from '@store/keyboard/context';
```

Comment:
some changes in this shouldn't be in this file, please remove them

Reply:
which changes are you trying to reflect, I am not getting

Reply:
There should be no changes in this file, these are the changes from your previous PR